docs/FAQ.md

```diff
@@ -37,6 +37,34 @@ Breaking large amounts of data up into manageable groups is generally a good ide
 
 No. VIAME Web and DIVE Desktop perform automatic transcoding if it is necessary.
 
+## How does video frame alignment work?
+
+When you annotate a video in DIVE, the true video is played in the browser using a native HTML5 video player.
+
+Web browsers report and control time in floating point seconds rather than using frame numbers, but annotations are created using frame numbers as their time indicators, so it's important to make sure these line up.
+
+Most of the time, videos are **downsampled** for annotation, meaning that the true video framerate (30hz, for example) is annotated at a lower rate, such as 5hz or 10hz.  Kwiver (the computer vision tool behind VIAME) uses a downsampling approach that sorts actual frames into downsampled buckets based on the start time of the frame.
+
+An implementation of this approach is described here.
+
+```python
+def get_frame_from_timestamp(timestamp, true_fps, downsample_fps):
+  downsampled_frame = timestamp * downsample_fps
+  real_frame = 
+
+  if downsample_fps >= true_fps:
+    # This is a true downsample
+    next_true_frame_boundary = Math.ceil(timestamp * true_fps)
+    return Math.floor(next_true_frame_boundary / downsample_fps)
+  
+  raise Exception('Real video framerate must be GTE downsample rate')
+```
+
+There are caveats with this approach.
+
+* It does not handle padding properly.  If a video begins or ends with padding, you may see a black screen in DIVE, but kwiver will wait for the first true frame to use as the representative for the bucket.
```

Comment:
I think if it ends with padding you may annotate a frame that shouldn't be used but that depends on if Kwiver uses this frame or not.  I'm going to check with Matt about this.